src/components/HeaderBreadcrumbs/HeaderBreadcrumbs.tsx

```diff
@@ -1,28 +1,37 @@
 import React from 'react';
+
 import {block} from '../../utils';
 import {HeaderBreadCrumbsProps} from '../../models';
-import {useMetrika} from '../../hooks/useMetrika';
+import HeaderBreadcrumbsItem from './HeaderBreadcrumbsItem';
 
 import './HeaderBreadcrumbs.scss';
 
 const b = block('header-breadcrumbs');
 
-export default function HeaderBreadcrumbs(props: HeaderBreadCrumbsProps) {
-    const {items, metrikaGoals, pixelEvents, theme = 'light', className} = props;
-    const handleMetrika = useMetrika();
+const COMPONENT_NAME = 'header-breadcrumbs';
 
-    const onClick = () => {
-        handleMetrika({metrikaGoals, pixelEvents});
-    };
+export default function HeaderBreadcrumbs(props: HeaderBreadCrumbsProps) {
+    const {
+        items,
+        metrikaGoals,
+        pixelEvents,
+        theme = 'light',
+        blockName = COMPONENT_NAME,
```

Comment:
COMPONENT_NAME sounds like inner code info and we can have many components with same name

Reply:
we changed `blockName` -> `context`
and we hid it in Context.Provider to avoid deep passing of props